src/test/java/com/google/cloud/spanner/pgadapter/ITJdbcTest.java

```diff
@@ -527,4 +527,80 @@ public void testCopyIn_Large_SucceedsWhenNonAtomic() throws SQLException, IOExce
       connection.createStatement().execute("delete from all_types");
     }
   }
+
+  @Test
+  public void testTwoDmlStatementsInSimpleMode() throws SQLException {
+    // This connection prefers the simple query mode. This makes the JDBC driver behave very similar
+    // to psql, and enables us to send multiple simple unparameterized statements in one batch.
+    try (Connection connection =
+        DriverManager.getConnection(
+            String.format(
+                "jdbc:postgresql://localhost:%d/?preferQueryMode=simple", server.getLocalPort()))) {
```

Comment:
Done. By the way, how do you run IT tests? I tried `mvn -DskipTests=false integration-test` but none was run.

Reply:
You need to use `-DskipITs=false` instead of `-DskipTests=false`.

`mvn -DskipITs=false integration-test` should work.
`mvn -DskipITs=false verify` should also work.

I normally run them directly from IntelliJ, as it gives me more control over which tests are run. Running all integration tests can take a long time.

Reply:
Thanks.